### DISCUSSION TITLE
Hetrograph

### DISCUSSION THREAD

**Discussion by hebahigazy** [pyg-team/pytorch_geometric#3227] [Q&A]
Hi,
I want to create data type of HetroData, but I receive error when I import the module:

My code:
from torch_geometric.data import HeteroData

Result:
ImportError: cannot import name 'HeteroData' from 'torch_geometric.data'

How I can fix this error?

**Comment by rusty1s** [comment]
Which version of `torch-geometric` are you using (`torch_geometric.__version__`)? Note that `HeteroData` was introduced in version >= 2.0.
